api/src/db/model/model.ts

```diff
@@ -3,13 +3,28 @@ import { logger } from '@bcgov/common-nodejs-utils';
 import { Pool } from 'pg';
 import { transformKeysToCamelCase } from '../utils';
 
-export default abstract class Model {
+export interface CommonFields {
+  id?: number,
```

Comment:
optional for ID?

Reply:
Ya. I did that because the ID will be present if it comes from the database but absent if it comes from a API request. I'll make a mental note and consider creating two different types if needed.